GitExtUtils/GitUI/Theming/AppColor.cs

```diff
@@ -1,5 +1,12 @@
 ﻿namespace GitExtUtils.GitUI.Theming
 {
+    /// <summary>
+    /// GitExtensions' application specific color names
+    /// </summary>
+    /// <remarks>
+    /// Values are stored in AppSettings class. Whenever new name is added here, add default value
+    /// to <see cref="AppColorDefaults"/> and \GitUI\Themes\win10default.colors
+    /// </remarks>
```

Comment:
Can we enforce it via a test of some kind?

Reply:
Great idea!

Reply:
done